classes/local/addvideo_form.php

```diff
@@ -296,4 +414,34 @@ protected function try_get_string($identifier, $component = '', $a = null) {
             return get_string($identifier, $component, $a);
         }
     }
+
+    /**
+     * Returns scheduled change visibility waiting time.
+     *
+     * @param int $ocinstanceid The opencast instance id.
+     * @param array $customminutes Custome minutes to be added or deducted on demand.
+     * @return int
+     */
+    protected function get_scheduled_change_visibility_waiting_time($ocinstanceid, $customminutes = []) {
+        $configwaitingtime = get_config('block_opencast', 'aclcontrolwaitingtime_' . $ocinstanceid);
+        if (empty($configwaitingtime)) {
+            $configwaitingtime = \block_opencast\local\visibility_helper::DEFAULT_WAITING_TIME;
+        }
+        $formtimestamp = $this->formtimestamp;
```

Comment:
I'm getting an undefined property error for `formtimestamp` when adding a video. Could you pls fix that?